libbeat/docs/newbeat.asciidoc

```diff
@@ -188,50 +238,28 @@ func (bt *Testbeat) Stop() {
 
 By default, the Beat object contains the following:
 
-- `done`: Channel used by the `Run()` function to stop when the `Stop()` function is called.
-- `config`: Configuration options for the Beat
-- `client`: Publisher that takes care of sending the events to the
+<1> `done`: Channel used by the `Run()` function to stop when the `Stop()` function is called.
+<2> `config`: Configuration options for the Beat
+<3> `client`: Publisher that takes care of sending the events to the
   defined output.
 
-The `Beat` parameter received by the `Run` method contains data about the
+The `Beat` parameter received by the `Run` function contains data about the
```

Comment:
OK, so I'll revert the change and fix the earlier mention where we call it a function. :-)